chroma_core/models/host_jobs/test_host_contact.py

```diff
@@ -108,9 +109,14 @@ def _test_yum_sanity(self, agent_ssh, auth_args, address):
             check_yum = """
 python -c "from yum import YumBase
 yb = YumBase()
+baselist = %s
+if len(yb.repos.listEnabled()) == 0 and len([x for x in yb.doPackageLists(pkgnarrow='installed', patterns=baselist)]) == len(baselist):
```

Comment:
Nit: they say `if not list:` will be more idiomatic Python than `if len(list) == 0`.
Nit2: if `baselist` contained wildcards, the lists might be different :)